lib/tasks/seed.rake

```diff
@@ -213,9 +213,9 @@ namespace :seed do
       Foreman::Logging.logger('permissions').level = Logger::ERROR
       Foreman::Logging.logger('audit').level = Logger::ERROR
       origin = ENV['origin'] || 'Puppet'
-      hosts = ENV['hosts'] || 10
-      reports = ENV['reports'] || 50
-      lines = ENV['lines'] || 100
+      hosts = (ENV['hosts'] || 10).to_i
+      reports = (ENV['reports'] || 50).to_i
+      lines = (ENV['lines'] || 100).to_i
```

Comment:
The seed rake task was simply broken, it would not accept any argument raising exception.